src/main/java/at/ac/tuwien/kr/alpha/common/predicates/ExternalNativePredicate.java

```diff
@@ -0,0 +1,36 @@
+package at.ac.tuwien.kr.alpha.common.predicates;
+
+import at.ac.tuwien.kr.alpha.common.terms.ConstantTerm;
+import at.ac.tuwien.kr.alpha.common.terms.Term;
+import at.ac.tuwien.kr.alpha.grounder.Substitution;
+
+import java.util.List;
+
+public class ExternalNativePredicate implements Predicate, Evaluable {
```

Comment:
What is native about this external predicate?

Reply:
This class wraps `java.util.function.Predicate` which I coined "native" predicates ... I was in a hurry.